packages/apollo-gateway/src/__tests__/integration/networkRequests.test.ts

```diff
@@ -353,7 +360,7 @@ describe('Downstream service health checks', () => {
       ).rejects.toThrowErrorMatchingInlineSnapshot(`"500: Internal Server Error"`);
     });
 
-    it('Rolls over to new schema when health check succeeds', async () => {
+    it.skip('Rolls over to new schema when health check succeeds', async () => {
```

Comment:
🥌

Reply:
See notes in bb83f17. We've decided to leave these skipped for now 😞